examples/BUILD

```diff
@@ -10,6 +10,10 @@ envoy_package()
 filegroup(
     name = "configs",
     srcs = [
+        "cors/backend/front-envoy.yaml",
```

Comment:
I would have expected https://github.com/envoyproxy/envoy/blob/master/test/config_test/example_configs_test.cc#L22 to need to change with these added configs, which makes me wonder if something is broken? Can you take a look and make sure these new configs are getting tested?

Reply:
Turns out this isn't broken but it doesn't account for multiple config files with the same name.

Fixed in https://github.com/envoyproxy/envoy/pull/5220